opentracing-shim/README.md

```diff
@@ -1,8 +1,10 @@
 [![Javadocs][javadoc-image]][javadoc-url]
-# OpenTelemetry - OpenTracing Shim
+# OpenTelemetry - OpenTracing Shim (Bridge)
```

Comment:
I'm not totally opposed to adding the parenthetical, but it's not the naming that we've been using on the project. Do you think this makes things more clear or discoverable?

Reply:
Yeah agree with @jkwatson - Shim is already a bridge so it seems like stuttering, I'd leave it out.

Reply:
Also worth pointing out that "bridge" has a different meaning in the instrumentation/agent project, so I'd avoid that here.